go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.19.1
 	sigs.k8s.io/jobset v0.5.2
```

Comment:
Could you specify main branch, and then let us check if it should work fine as expected.

Reply:
Sure, I assumed you preferred to wait until the upcoming JobSet release :).

Reply:
@tenzen-y @astefanutti Should we merge this PR once JobSet release v0.8.0 ?
I think, we are planning to release it this week: https://github.com/kubernetes-sigs/jobset/issues/774

cc @kannon92 @ahg-g

Reply:
I've replaced the JobSet API version to v0.8.0-devel for now.

I'm working on adapting the unit tests but the main controller should work fine.